rest/serializers.py

```diff
@@ -97,3 +103,49 @@ class Meta:
             'id',
             'slug',
         )
+
+
+class GroupSerializer(Serializer):
+    """
+    Serializer for base_group_type
+    """
+    # pylint: disable=abstract-method
```

Comment:
This seems like something we could disable globally. It's not the kind of thing we're going to do by accident. I think it's one of those pylint warnings that's overzealous.